tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -95,6 +97,41 @@ class WskBasicTests
             stderr should include(errormsg)
     }
 
+    it should "accept a 3 part Fully Qualified Name without a leading '/'" in {
+        val packageName = "packageName"
```

Comment:
I recommend using labels that can uniquely identify this particular test.  If there's a need to traverse the backend logs, having unique labels is helpful